src/kibana/plugins/visualize/editor/agg.html

```diff
@@ -54,7 +54,7 @@
 
       <!-- remove button -->
       <button
-        ng-if="stats.count > stats.min"
+        ng-if="canRemove(agg)"
```

Comment:
`stats.count` is the count of all the aggs, not of all of the aggs of that particular type.
`canRemove()` essentially is just a reduce to make sure there's always the minimum of a particular agg.